actions/archive2disk/v1/pkg/archive/archive.go

```diff
@@ -51,61 +52,17 @@ func Write(archiveURL, archiveType, path string) error {
 		// defer zipOUT.Close()
 		// out = zipOUT
```

Comment:
I know this is not part of your changes but commenting on everything in "gz" means not supported? Maybe add Println message or remove gz?